compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -3923,6 +3923,29 @@ TR::AMD64RegImm64SymInstruction::autoSetReloKind()
 // Generate methods
 ////////////////////////////////////////////////////////////////////////////////
 
+TR::Instruction* generateBreakOnDFSet(TR::CodeGenerator *cg, TR::Instruction* cursor)
+   {
+   if (!cursor)
+      cursor = cg->getAppendInstruction();
+
+   TR::RealRegister *espReal = cg->machine()->getRealRegister(TR::RealRegister::esp);
+   cursor = generateInstruction(cursor, PUSHFD, cg);
+   TR::LabelSymbol* begLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol* endLabel = generateLabelSymbol(cg);
+   begLabel->setStartInternalControlFlow();
+   endLabel->setEndInternalControlFlow();
+
+#define DF_MASK 0x400
```

Comment:
rather than a macro that will escape the scope should this just be a local constant? I worry about the macro escaping elsewhere or doing something else strange.